### DISCUSSION TITLE
MessageSent event

### DISCUSSION THREAD

**Discussion by iZeaoGamer** [JaxkDev/DiscordBot#46] [Q&A] [answered]
So hey. I have my own bot on discord with this plugin, and I was wondering - Do the events like MessageSent Event, work with bots too? For example, if the bot types smth in chat, like a command, etc, would it fire the MessageSent Event?

**Comment by iZeaoGamer** [comment]
So basically, when the bot chats on discord, does the MessageSent event fire?

**Comment by JaxkDev** [comment, marked as answer]
No, not if it was the bot that sent the message.
